source/server/core_server.cpp

```diff
@@ -238,6 +258,22 @@ int main(int argc, char** argv)
 #endif
 #if defined(_WIN32)
   nidevice_grpc::set_console_ctrl_handler(&StopServer);
+#else
+    SysFsWrite("/dev/cgroup/cpuset/system_set/cpus", "0-5");
```

Comment:
@ccifra I think this code is specific to the RT use case but we would be executing this even for linux desktop. I haven't seen a side effect on linux desktop but is it something I need to think about regarding the behavior on RT vs Linux Desktop?
@astarche

Reply:
Can you put this behind a toggle check?

Reply:
My bad, I had it locally but forgot to push the change.